python/whylogs/api/logger/experimental/logger/actor/data_logger.py

```diff
@@ -1,8 +1,17 @@
 from abc import abstractmethod
 from typing import Any, Dict, Generic, List, Optional, TypeVar, Union
 
-from whylogs.core.stubs import pd
+try:
```

Comment:
Why do we not like stubs?

Reply:
Pyright (apparently rightfully) calls them out as invalid python because they aren't types, they're values, so they can't be used in all of the places that types are used. I had to either silence the pyright errors or ditch the stubs. Something is definitely fishy with the stubs though, I'm not sure this is the best way of getting around a hard dependency on pandas.